src/actions/route_helpers.cr

```diff
@@ -78,3 +78,5 @@ module DppmRestApi::Actions::RouteHelpers
     ).tap &.ensure_pkg_dir
   end
 end
+
+require "./route_helpers/**"
```

Comment:
```suggestion
require "./route_helpers/*"
```
Not a fan of `**`, which import every sub-files.
Better to only use `*`, this limit issues related to file require order.